src/modules/Microsoft.Resources/deploymentScripts/version.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Since we're breaking free from CARML, do we care about this file? I'm fine either way. We can also defer the cleanup until we deal with it for the other files. Just asking what others think...